README.md

```diff
@@ -11,7 +11,7 @@ for details on getting up and running with your very own robot friend.
 In most cases, you'll probably never have to hack on this repo directly if you
 are building your own bot. But if you do, check out [CONTRIBUTING.md](CONTRIBUTING.md)
 
-If you'd like to chat with Hubot users and developers, drop by #hubot on FreeNode IRC.
+If you'd like to chat with Hubot users and developers, [join us on Slack](https://hubot-slackin.herokuapp.com/).
```

Comment:
I agree, it's simpler with just slack. Still we would need to regain control on the irc channel to setup a link to slack on the `/topic`